lib/uri.py

```diff
@@ -0,0 +1,52 @@
+from .. import config as _config
```

Comment:
`__Type` snap ik niet goed. Voor `__Resource` begrijp ik het doel van de abstractie ... waarschijnlijk is een [bestaande, veelvuldig gebruikte library](https://rdflib.readthedocs.io/en/stable/apidocs/rdflib.html#module-rdflib.resource) wel een beter pad als je deze abstracties wil.

Reply:
je hoeft dit niet meteen te veranderen, maar besef dat deze zelfgemaakte abstracties steeds wat extra leeswerk kosten. Het is dus altijd wat af te wegen of deze de moeite waard zijn.